conda-store-server/conda_store_server/action/generate_constructor_installer.py

```diff
@@ -0,0 +1,127 @@
+import os
+import pathlib
+import subprocess
+import sys
+import tempfile
+import warnings
+
+import yaml
+from conda_store_server import action, schema
+
+
+def get_installer_platform():
+    # This is how the default platform name is generated internally by
+    # constructor. For example: osx-arm64, linux-64, win-64.
+    # https://github.com/conda/constructor/blob/main/CONSTRUCT.md#available-platforms
+    from conda.base.context import context
+
+    return context.subdir
```

Comment:
Note the value of `context.subdir` can be overridden by the `CONDA_SUBDIR` env var. `context.subdir` will assume the OS and architecture of whatever Python `conda` is running on by default, but it can also be overridden by its `--platform` flag.

I assume this function is running on the same server so we are not offering any kind of cross-building or cross-solving yet. However, we should add a note that this is cross-friendly because at some point we will want to tackle that feature, I assume.

xref https://github.com/conda-incubator/conda-store/issues/71#issuecomment-1790332315